<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrto</class>
 <widget class="QWidget" name="CreateOrto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxExtraerOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Extreure Orto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDatesOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete dates</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonFolderOutOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Directori desti</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFolderOutOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_32">
     <item>
      <widget class="QLabel" name="labelAmbitoProyectoOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zone project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAmbitoProyectoOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_33">
     <item>
      <widget class="QLabel" name="labelTamanoPixelOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pixel size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTamanoPixelOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_34">
     <item>
      <widget class="QLabel" name="labelCoordinateSystemOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Coordinate system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCoordinateSystemOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBoxCortarOrto">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cut DTM</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_35">
     <item>
      <widget class="QLabel" name="labelCortarOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Size cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTamanyoCortarOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_36">
     <item>
      <widget class="QLabel" name="labelNumeroCanalesspasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Numbers chanels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxNumeroCanalesspasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxFootPrintMaskOrto">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>FootPrintMask</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_37">
     <item>
      <widget class="QLabel" name="labelSelectSensor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Select sensor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSelectSensor">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_38">
     <item>
      <widget class="QLabel" name="labelOffsetPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Offset flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOffsetPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_39">
     <item>
      <widget class="QLabel" name="labelAnchoPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>width flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAnchoPasadaOrto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
